doc/sphinx/install/macos.md

```diff
@@ -1,41 +0,0 @@
-(sec-install-macos)=
-# macOS Matlab Toolbox
-
-The Cantera Matlab toolbox can be installed using a macOS-specific installer. The
-toolbox requires macOS version 10.15 (Catalina) or higher and a copy of Matlab built for
-Intel processors (which will run under Rosetta 2 on ARM-based processors).
```

Comment:
Should this redirect users to the 3.0 package, as you've done for the conda and Windows installer pages?

Reply:
As the macOS install instructions are specific to the legacy MATLAB toolbox which this PR removes, it in effect eliminates the need for the installer (which would be one less item to maintain) - from that perspective, the macOS installation instructions are moot? (the only instance of`sec-install-macos` I found was in connection with MATLAB).

I did, on the other hand, add an admonition on the main install page that the legacy Matlab toolbox was removed, which would be my preferred resolution. Please let me know in case there's another use of the macOS installer that I am unaware.

Reply:
Yes, the macOS package was only for the Matlab toolbox. From the standpoint of a Mac user wanting to install the Matlab toolbox, though, they need to see the directions to use the 3.0 toolbox somewhere. I think putting that information on the installation index page satisfies that scenario.